Doc/library/sys.rst

```diff
@@ -248,7 +248,11 @@ always available.
    before the program exits.  The handling of such top-level exceptions can be
    customized by assigning another three-argument function to ``sys.excepthook``.
 
-   See also :func:`unraisablehook` which handles unraisable exceptions.
+    .. seealso::
+
+       The :func:`sys.unraisablehook` function handles unraisable exceptions
+       and the :func:`threading.excepthook` function handles uncaught
+       :func:`threading.Thread.run` exception.
```

Comment:
Grammar

Reply:
English is not my first language, so if you spot something, please suggest a fix :-)

Reply:
```suggestion
       :func:`threading.Thread.run` exceptions.
```

Reply:
```
the uncaught
       :func:`threading.Thread.run` exception.
```